TDS_2/doc/TDS_2/Concepts/TriangulationDataStructure_2.h

```diff
@@ -110,26 +110,6 @@ Handle to a face.
 */ 
 typedef unspecified_type Face_handle; 
 
-/*!
-This template class allows to get the type of a triangulation 
-data structure that only changes the vertex type. It has to define a type 
-`Rebind_vertex<Vb2>::%Other` which is a <I>rebound</I> triangulation data structure, that is, the 
-one whose `TriangulationDSVertexBase_2` will be `Vb2`.
-\note It can be implemented using a nested template class.
-*/ 
-template <typename Vb2> 
-using Rebind_vertex = unspecified_type;
```

Comment:
The rebind mechanism is required because CGAL::Triangulation_data_structure (the model) is templated by vertex/cell base classes, with those being themselves templated by the TDS. So it's a requirement of the model (and is as such properly documented in the concepts of vertex/face/cell classes that are used to template CGAL::Triangulation_data_structure), but it's not a requirement of the concept TDS.